enterprise/server/scheduling/task_router/task_router.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 var (
-	affinityRoutingPermitted = flag.Bool("executor.affinity_routing_permitted", true, "If set, along with the 'affinity-routing' platform property, then (experimental) affinity routing is used. If false, then affinity routing is never used. This flag is intended to be used as an emergency shut-off for this experimental feature.")
+	affinityRoutingEnabled = flag.Bool("executor.affinity_routing_enabled", true, "Enables affinity routing, which attempts to route actions to the executor that most recently ran that action. This flag is intended for disabling this feature in an emergency.")
```

Comment:
i don't think `This flag is intended for disabling this feature in an emergency.` adds much but up to you :P